sdp-explainer.md

```diff
@@ -100,18 +100,18 @@ readable.pipeThrough(new TransformStream(
 RTCRtpReceiver.AddCodecCapability(customCodec);
 pc.ontrack = (receiver) => {
 
-   for (codec in receiver.getParameters.codecs) {
+   for (codec in receiver.getParameters().codecs) {
       if (codec.mimeType == “application/x-encrypted”) {
          encryptedPT = codec.payloadType;
       }
    }
-   receiver.addDecodeCapability({mimeType: video/vp8, payloadType=208});
+   receiver.addDecodeCapability({mimeType: video/vp8, payloadType=encryptedPT});
```

Comment:
actually not encryptedPT. We could add a constant saying decryptedPT, or negotiate it as part of the SDP, but it is the PT that is assigned after the transform, and used by RTCRTPReceiver to choose the right decoder, not the PT that is used on the wire.